### PR TITLE
[BLAS][portBLAS] Update interface call for rotmg for portBLAS backend

### PR DESCRIPTION
**Pull request by s-Nick**
# Description

Currently the dependencies on copy event isn't always respected with OpenCL on CPU devices, this cause random failure with this operator.
To reproduce the failure, run `RotmgUsm` operator in a loop using OpenCL CPU portBLAS backend, it will eventually fail, it generally takes less that 10 iterations.
Due to internal changes in portBLAS the copies are not necessary anymore.

Possible related issue https://github.com/intel/llvm/issues/14623

# Checklist

## All Submissions

- [x] Do all unit tests pass locally? Attach a log.
[rotmg_amd_log.txt](https://github.com/user-attachments/files/16438758/rotmg_amd_log.txt)
[rotmg_cpu_log.txt](https://github.com/user-attachments/files/16438760/rotmg_cpu_log.txt)
[rotmg_nvidia_log.txt](https://github.com/user-attachments/files/16438762/rotmg_nvidia_log.txt)
[rotmg_pvc_log.txt](https://github.com/user-attachments/files/16438764/rotmg_pvc_log.txt)

- [x] Have you formatted the code using clang-format?

## Bug fixes

- [x] Have you included information on how to reproduce the issue (either in a
      GitHub issue or in this PR)?

### CODE REVIEW

**Comment by hjabird** [comment]
Looks good to me!

* Perhaps add a link to the portBLAS PR in the description.
* I'll approve once the portBLAS PR has merged.

**Review by pgorlani — APPROVED** [review]

**Review by Rbiessy — APPROVED** [review]

**Pull request merged**
